server/controllers/chartsController.js

```diff
@@ -2,6 +2,8 @@ import { ObjectID } from 'mongodb'
 
 import { collections } from '../utils/mongoCollections'
 
+const Totals = 'Totals'
```

Comment:
can we be a little more descriptive here and also use all caps for the constant?

```suggestion
const TOTALS_STUDY = 'Totals'
```